bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/i18n/awattar_de.properties

```diff
@@ -0,0 +1,166 @@
+binding.awattar.name = aWATTar Binding
```

Comment:
@lolodomo Shall this be part of the binding?

Reply:
No, it is better to translate in Crowdin. 
But if the translation was fully done the way, as it is a new binding, an exception could be done : just after the merge, someone should take care to load the German transitions in Crowdin.

Reply:
I removed the name from the properties file

Reply:
It was about the entire file. As you already did all translations in the properties file, I suggest you to add the name property again and after this has been merged, you ask a maintainer to load your whole translation into Crowdin.

Reply:
@lolodomo @jlaur can you load the German translations into Crowdin?